drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/AttributeColumnPageView.java

```diff
@@ -59,7 +62,12 @@ public void init(final AttributeColumnPage page) {
 
     @EventHandler("attributeList")
     public void onSelectAttribute(ChangeEvent event) {
```

Comment:
Is event parameter is really needed, it just does not have any usage in the function

Reply:
Yes, it is feature of GWT, the `event` Parameter is needed even if it is not used, not sure if pointing to correct documentation: http://google.github.io/gwteventbinder/

Reply:
ok